### DISCUSSION TITLE
An issue to say thank you

### DISCUSSION THREAD

**Discussion by SirMishaa** [MyOutDeskLLC/Tailwind-Formatter#45] [General]
I wanted to say thank you for your work, I use Tailwind every day and this amazing plugin helps me a lot. 

That's the strength of open source! :p

**Comment by tance77** [comment]
I'm glad you are enjoying the open source plugin. Thank you for your kind words. They have brightened up our day. :)

**Reply by hms** [reply]
One more voice to the chorus of people who loved this plugin and really missed it when you thought prettier was going to work as an equivalent solution. 

My code base turned into a tailwind mess on that day. Your taking on the challenge and time hit to bring this back truly made my day. 

Thank you. Thank you. Thank you!
